static/js/floating_recipient_bar.js

```diff
@@ -10,9 +10,7 @@ let is_floating_recipient_bar_showing = false;
 
 function top_offset(elem) {
     return (
-        elem.offset().top -
-        $("#message_view_header").safeOuterHeight() -
-        $("#navbar_alerts_wrapper").height()
+        elem.offset().top - $("#navbar-middle").offset().top - $("#navbar-middle").safeOuterHeight()
```

Comment:
Why use `$("#navbar-middle").offset().top`? We should use `$(window).scrollTop()` to simplify understanding of the change.

Reply:
This was done based on this review comment : https://github.com/zulip/zulip/pull/16411#discussion_r494586964